### PR TITLE
Remove unnecessary `else` statements

### PR DESCRIPTION
**Pull request by distantnative**
### Refactoring
- Removed unnecessary else statement where it helps to improve readability of the code

## Ready?

- [x] Unit tests for fixed bug/feature
- [x] In-code documentation (wherever needed)
- [x] Tests and checks all pass

### For review team

- [ ] Add changes to release notes draft in Notion
- [x] ~~Add to [website docs release checklist](https://github.com/getkirby/getkirby.com/pulls) (if needed)~~

### CODE REVIEW

**Review by afbora — DISMISSED** [review]

File: src/Cms/Content.php
Comment (lukasbestle) [inline src/Cms/Content.php:118]: TBH I think this is less clear now. Previously you could see on the first glance that this `if` only applies if there is an `$oldField`. Now you need to spot that question mark.

File: src/Cms/System.php
Comment (lukasbestle) [inline src/Cms/System.php:325]: Here I like it on the other hand because it really makes the code a lot shorter and increases the overview.

File: src/Database/Query.php
Comment (lukasbestle) [inline src/Database/Query.php]: Double newline:

```suggestion

```

File: src/Cms/Content.php
Comment (distantnative) [reply src/Cms/Content.php:118]: The check s there to see if old and new field have the same type. I think it's rather clear that if oldField does not exist the type wouldn't be the same even if we don't first check explicitly for oldField on its own.

**Comment by distantnative** [comment]
@lukasbestle is this ok to be merged then? Needs an approval then, can't merge without. thanks!

**Comment by bastianallgeier** [comment]
@distantnative the code cov issue is really just about rediscovering lines that are as uncovered as they were before, right? Otherwise it looks good to me. Feels nice and clean.

**Review by bastianallgeier — APPROVED** [review]

**Comment by distantnative** [comment]
@bastianallgeier yes that's my understanding. So I think it's one of the cases where manual review can replace that check.

**Comment by lukasbestle** [comment]
Yep.

**Pull request merged**
